### PR TITLE
Feat: REPLAT-5155 In depth header width

### PR DESCRIPTION
**Pull request by jimhunty**
https://nidigitalsolutions.jira.com/browse/REPLAT-5155

* Sets a max width of 1180px for items that are considered "100%" width.
* Increases in font size for Headline
* Meta information changes display layout and introduces a seperator

### CODE REVIEW

File: packages/article-in-depth/__tests__/shared-with-style.native.js [hunk 1]
```diff
@@ -177,4 +178,37 @@ export default () => {
 
     expect(testRenderer).toMatchSnapshot();
   });
+
+  it("tablet full article with style in the culture magazine", () => {
```
Comment (tuncaulubilge) [inline packages/article-in-depth/__tests__/shared-with-style.native.js]: Do we need all three styles tested in tablet?

Comment (nathggns) [reply packages/article-in-depth/__tests__/shared-with-style.native.js]: I don't think so. I didn't do that on magazine.

Comment (jimhunty) [reply packages/article-in-depth/__tests__/shared-with-style.native.js]: The styling is different on tablet, so I think we do. We want to track that this doesn't regress.

Comment (nathggns) [reply packages/article-in-depth/__tests__/shared-with-style.native.js]: The tablet-specific styles are not dependent on the section, as far as I'm aware?

Comment (jimhunty) [reply packages/article-in-depth/__tests__/shared-with-style.native.js]: They are, the font changes and therefore the font matrix (font, font size, line height) for that changes based on the section

**Pull request merged**